src/dotnet/APIView/APIViewWeb/wwwroot/js/comments.ts

```diff
@@ -147,4 +181,27 @@
         $(commentBox).replaceWith(partialViewResult);
         return false;
     }
+
+    function toggleAllCommentsAndDiagnosticsVisibility(show: boolean) {
+        $(SEL_COMMENT_ROW).toggle(show);
```

Comment:
This can be rewriten as something like
```
$("[data-line-id]").foreach(function () {
	var lineId = getLineId(this);

	 getCommentsRow(id).toggle(show);
     getDiagnosticsRow(id).toggle(show);
	 tooggleCommentIcon(id, show);
});
```

And avoid all the complicated selector navigation.